internal/providers/azure/azure.go

```diff
@@ -100,62 +107,7 @@ func FetchFromOvfDevice(f *resource.Fetcher, ovfFsTypes []string) (types.Config,
 	logger.Debug("reading config")
 	rawConfig, err := ioutil.ReadFile(filepath.Join(mnt, configPath))
 	if err != nil && !os.IsNotExist(err) {
```

Comment:
How do we distinguish between 1) the user not specifying an Ignition config, and 2) this not being the config drive?  Can we detect the config drive by looking for a second file which is always present?